aicsimageio/readers/ome_zarr_reader.py

```diff
@@ -135,6 +157,26 @@ def _get_coords(
 
         return coords
 
+    @staticmethod
+    def _get_pixel_size(
+        reader: ZarrReader, dims: List[str], series_index: int, resolution_index: int
+    ) -> Tuple[Optional[float], Optional[float], Optional[float]]:
+
+        coord_transform = reader.root_attrs["multiscales"][series_index]["datasets"][
+            resolution_index
+        ]["coordinateTransformations"]
+
+        spatial_coeffs = {}
+
+        for dim in ["X", "Y", "Z"]:
```

Comment:
Should we use the constants for these here? `DimensionNames.SpatialX` for example

Reply:
resolved in [5d16d65](https://github.com/AllenCellModeling/aicsimageio/pull/509/commits/5d16d655856ed02601fd3a3b6084b96a8e008137)